spring-cloud-alibaba-examples/nacos-example/nacos-config-example/src/main/resources/bootstrap.properties

```diff
@@ -18,18 +18,16 @@ spring.cloud.nacos.password=nacos
 #spring.cloud.nacos.config.shared-data-ids=common.properties,base-common.properties
 
 ## recommended.
-spring.cloud.nacos.config.shared-configs[0].data-id=test2.yaml
+spring.cloud.nacos.config.shared-configs[0].data-id=data-source.yaml
 spring.cloud.nacos.config.shared-configs[0].refresh=true
 ## the default value is 'DEFAULT_GROUP' , if not specified.
-spring.cloud.nacos.config.shared-configs[0].group=GROUP_APP1
+#spring.cloud.nacos.config.shared-configs[0].group=GROUP_APP1
 
 ## not recommended.
 #spring.cloud.nacos.config.ext-config[0]=ext.properties
 ## recommended.
-spring.cloud.nacos.config.extension-configs[0].data-id=extension1.properties
+spring.cloud.nacos.config.extension-configs[0].data-id=ext-data-source.yaml
```

Comment:
I think if we use two apps show how to use extension-configs is better, one has extension-configs configuration, the other one does not